crud/src/main/java/org/openmbee/mms/crud/services/NodeGetHelper.java

```diff
@@ -13,14 +13,24 @@
 import org.openmbee.mms.data.domains.scoped.Branch;
 import org.openmbee.mms.data.domains.scoped.Commit;
 import org.openmbee.mms.data.domains.scoped.Node;
+import org.openmbee.mms.core.dao.CommitIndexDAO;
+import org.openmbee.mms.json.CommitJson;
 import org.openmbee.mms.json.ElementJson;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 import static org.openmbee.mms.core.config.ContextHolder.getContext;
 
 @Service
 public class NodeGetHelper extends NodeOperation {
 
+    protected CommitIndexDAO commitIndex;
```

Comment:
:(

Reply:
it'll get refactored when we merge in to the boeing branch anyways